packages/web/src/js/element/region.ts

```diff
@@ -113,6 +112,41 @@ export const oddInfo: ElementInfo = {
     },
 };
 
+export const columnIndexerInfo: ElementInfo = {
+    getInputHandler(ref: StateRef, grid: Grid, svg: SVGSVGElement): InputHandler {
+        return getInputHandler(ref, grid, svg, '');
+    },
+    order: 42,
+    inGlobalMenu: false,
+    menu: {
+        type: 'select',
+        name: 'Column Indexer',
+        icon: 'odd-even',
```

Comment:
Just want to double check - you're intentionally using the odd/even icon, right? (I don't think we have an icon for 159 anyway)

Reply:
Yes, because I didn't know what else to use.

Reply:
Makes sense! We can always update it later when we have an icon specifically for indexers.